std/assembly/internal/itoa.ts

```diff
@@ -67,7 +67,7 @@ function DIGITS(): u32[] {
   return table; // inlines to a constant memory offset
 }
 
-// Count number of decimals in value
+// General routine for count number of decimals in value
 export function decimalCount<T>(value: T): i32 {
```

Comment:
Are you also planning to make this one a wrapper around decimalCountU32/U64?

Reply:
Did not quite understand what you mean? `decimalCount<T>` is not using for now, I leave it for external usage only

Reply:
Yeah, but let's say someone would use the generic one, then it would still compile once for each type of type parameter. Now, if it would reuse the more specific 32/64 ones, that code would be common, while the generic wrapper would just compile to a minimal wasm function calling the specific one.

Reply:
I don't think split it to four separate concreate versions is good idea (mean i32, u32, i64, u64)

Reply:
If someone want utilize and compile with optimal code size it should use `decimalCoun<T = i32 | u32 | i64 | u64>`

Reply:
Within `decimalCount<T>`, the function could just statically check for `sizeof<T>`, call the respective specific one, i.e. decimalCountU32, and if `isSigned<T>` also handle signage before doing so, resulting in a minimal generic wrapper that works for all sizes of integers.

Reply:
now I see, yep I can improve this but in our case `decimalCountU64` has some special optimization which possible only for current implementation of itoa another word it depends on context. For example I skip some checks which unnecessary for our case. So I can't reuse  new routines `decimalCountU32/decimalCountU64` for `decimalCount<T>`

Reply:
Or if you mean something else you are free to improvements after merge =)

Reply:
I see. Well, in this case we might just consider removing the generic one I guess.

Reply:
ok